### DISCUSSION TITLE
Can't sign in

### DISCUSSION THREAD

**Discussion by DEVooooo** [INGInious/INGInious#999] [Q&A] [answered]
I registered to https://inginious.org and used the exact information to log in but it didn't work. I have tried so many times and changed the password but got the same result! 
What should I do next? 
Thank you

**Comment by anthonygego** [comment, marked as answer]
Hi,

Can you tell us your INGInious username so that we can investigate ?

Thanks.

**Reply by DEVooooo** [reply]
Hey, 
It was my bad. I thought I could login using the email address. when I tried with the username it worked.
Thank you so much
